helpers/controller.go

```diff
@@ -1,17 +1,31 @@
 package helpers
 
 import (
+	"google-scraper/models"
+
 	"github.com/beego/beego/v2/server/web"
 	"github.com/iancoleman/strcase"
 )
 
-func SetDataAttributes(c *web.Controller) {
+func SetDataAttributes(c *web.Controller, user *models.User) {
 	controllerName, actionName := c.GetControllerAndAction()
 
 	c.Data["ControllerName"] = strcase.ToSnake(controllerName)
-	c.Data["ActionName"] =  strcase.ToSnake(actionName)
-	c.Data["AlertMap"] = map[string] string {
+	c.Data["ActionName"] = strcase.ToSnake(actionName)
+	c.Data["CurrentUser"] = user
+	c.Data["navbarExpandModifierCssClass"] = navbarExpandModifierCssClass(user)
+	c.Data["AlertMap"] = map[string]string{
 		"success": "success",
-		"error": "danger",
+		"error":   "danger",
+	}
+}
+
+func navbarExpandModifierCssClass(user *models.User) string {
```

Comment:
Having complex logic for a class name does not seem the right way to implement the layout 🤔 It required code in the controller, view and CSS styles 😓 Can you think fo an alternative even if it requires to change the UI?

Reply:
@olivierobert I updated it by using two navbar partial(for guest user and autheticated user) in here https://github.com/junan/google-scraper/pull/27/commits/4132290e3b66006051c700ed9ff4aad6bf44e943

Reply:
Simpler and easier to maintain 👏